test/e2e/util/k8s/secret.go

```diff
@@ -72,3 +72,25 @@ func WaitForSecretsComplete(c clientset.Interface, ns, secretName string) error
 func GetSecret(c clientset.Interface, ns, secretName string) (*v1.Secret, error) {
 	return c.CoreV1().Secrets(ns).Get(context.TODO(), secretName, metav1.GetOptions{})
 }
+
+// CreateVCCredentialSecret refer to https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/v1.3.0/docs/vanilla.md
```

Comment:
Is it related to golang bump up, how is it called?

Reply:
No. This is not needed, and it's introduced in resolving conflict. Removed.